src/errors.rs

```diff
@@ -1,10 +1,10 @@
 #![allow(clippy::large_enum_variant)]
 #![allow(deprecated)] // because of `Error::description` deprecation in `error_chain`
 
-use crate::component_for_bin;
 use crate::dist::dist::Profile;
 use crate::dist::manifest::{Component, Manifest};
 use crate::dist::temp;
+use crate::{component_for_bin, Toolchain};
```

Comment:
Why was this reordered?

Reply:
Oops, this is left over from my failed attempt to make `Toolchain` part of `BinaryNotFound`. I reverted the change.

Reply:
Hmm, it looks like rustfmt really wants me to re-order it:
```
Run cargo fmt --all -- --check
Diff in /home/runner/work/rustup/rustup/src/errors.rs at line 1:
 #![allow(clippy::large_enum_variant)]
 #![allow(deprecated)] // because of `Error::description` deprecation in `error_chain`
 
-use crate::{component_for_bin, Toolchain};
 use crate::dist::dist::Profile;
 use crate::dist::manifest::{Component, Manifest};
 use crate::dist::temp;
Diff in /home/runner/work/rustup/rustup/src/errors.rs at line 8:
+use crate::{component_for_bin, Toolchain};
 use error_chain::error_chain;
 use std::ffi::OsString;
 use std::io::{self, Write};
##[error]Process completed with exit code 1.
```
Seems like a rustfmt bug? Adding a `{` shouldn't change the order.

Reply:
I guess `{` sorts after `d` for rustfmt :(